Tesla.css

```diff
@@ -0,0 +1,477 @@
+* {
+  margin: 0;
+}
+
+body {
+  max-width: 1920px;
+  margin: 0 auto;
+  font-family: Montserrat, sans-serif;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.header {
+  position: relative;
+}
+
+.logo {
+  width: 166px;
+  height: 51px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
+}
+
+.nav {
+  width: 93%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  margin-top: 24px;
+}
+
+.nav__link {
+  font-size: 15px;
+  white-space: nowrap;
+  margin: 0 10px;
+}
+
+.nav__model {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 37%;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+
+.nav__input {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 14%;
+}
+
+.nav__link:hover {
+  color: #E51937;
+}
+
+.nav__link-s::after {
+  content:'';
+  display: block;
+  width: 100%;
+  height: 2px;
+  background: #E51937;
+  margin-top: 6px;
+}
+
+ /* .nav.nav__link:hover::after {
+  content:'';
+  display: block;
+  width: 60px;
+  height: 2px;
+  background: #E51937;
+  margin-top: 6px;
+} */
+
+.background {
+  background: url(img/background-header.png);
+  background-repeat: no-repeat;
+  max-width: 1514px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)